src/editors/sharedComponents/Feedback/FeedbackBox.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
```

Comment:
This component (and all the other feedback related files) is only found in the problem editor so it should go in the AnswerWidget folder

Reply:
why? It could be used by other components this way, and there is nothing really specific to the answerwidget in there.